src/Raven.Server/NotificationCenter/NotificationsStorage.cs

```diff
@@ -17,23 +18,16 @@
 
 namespace Raven.Server.NotificationCenter
 {
-    public sealed unsafe class NotificationsStorage
+    public sealed unsafe class NotificationsStorage(ServerStore serverStore, string resourceName = null)
     {
-        private readonly string _tableName;
+        private readonly string _tableName = GetTableName(resourceName);
 
-        private readonly Logger Logger;
+        private readonly Logger Logger = LoggingSource.Instance.GetLogger<NotificationsStorage>(resourceName);
 
         private StorageEnvironment _environment;
 
         private TransactionContextPool _contextPool;
 
-        public NotificationsStorage(string resourceName = null)
```

Comment:
avoid doing this in a non-latest branch (7.0 as of today), usually this is a nightmare during merge, keep it for now, just a tip for future